config/playground/eks/dev/workstation-external-ip.tf

```diff
@@ -0,0 +1,18 @@
+#
+# Workstation External IP
+#
+# This configuration is not required and is
+# only provided as an example to easily fetch
+# the external IP of your local workstation to
+# configure inbound EC2 Security Group access
+# to the Kubernetes cluster.
+#
+
+data "http" "workstation-external-ip" {
+  url = "http://ipv4.icanhazip.com"
```

Comment:
any reason why not simply `ifconfig` command?

Reply:
There is no special requirement for ipv4.icanhazip.com, it is recommended by terraform aws provider.